src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -475,64 +454,6 @@ mocha.describe('manage_nsfs cli', function() {
             assert_account(account, account_options);
         });
 
-        mocha.it('cli account update name by access key', async function() {
-            const action = nc_nsfs_manage_actions.UPDATE;
-            const update_options = {
-                config_root,
-                new_name: 'account1_new_name',
-                access_key: account_options.access_key,
-                secret_key: account_options.secret_key,
-            };
-            account_options.new_name = 'account1_new_name';
-            const update_response = await exec_manage_cli(type, action, update_options);
-            updating_options = { ...updating_options, name: update_options.new_name };
-            assert_response(action, type, update_response, updating_options);
-            account_options = { ...account_options, new_name: undefined, name: update_options.new_name };
-            const account_symlink = await read_config_file(config_root, access_keys_schema_dir, access_key, true);
-            assert_account(account_symlink, account_options);
-            const account = await read_config_file(config_root, accounts_schema_dir, account_options.name);
-            assert_account(account, account_options);
-        });
-
-        mocha.it('cli account update access key by name', async function() {
```

Comment:
please instead of removing this test just replace new_access_key with access_key

Reply:
I was not removed It was moved into the jest tests. under `cli update account access_key and secret_key using flags`